reference/docs-conceptual/whats-new/What-s-New-in-PowerShell-Core-60.md

```diff
@@ -0,0 +1,345 @@
+# What's New in PowerShell Core 6.0
+
+## Renamed `powershell(.exe)` to `pwsh(.exe)`
```

Comment:
Maybe change heading to be more about Windows PowerShell and PowerShell Core 6 side-by-side and this decision was made, in part, to support this?